trie/typing.py

```diff
@@ -57,9 +58,15 @@ def __repr__(self):
         return hex(self.value)
 
 
+# A user-input value, where each element will be validated as a Nibble instead of int
+NibblesInput = Sequence[int]
+
+
 class Nibbles(tuple):
```

Comment:
I think the should be `class Nibbles(Tuple[int, ...])` for more concrete typing.